pkg/table/table.go

```diff
@@ -138,12 +145,24 @@ func (m *Model) SetRows(rows []Row) tea.Cmd {
 }
 
 func (m *Model) updateFilteredRows() {
+	rows := m.rows
+	if filterText := m.filterText(); filterText != "" {
+		rows = make([]Row, 0)
+		for _, row := range m.rows {
+			for _, field := range row.Fields {
+				if s, ok := field.(string); ok && strings.Contains(s, filterText) {
+					rows = append(rows, row)
+				}
+			}
+		}
```

Comment:
Suggest moving this nested for loop into the other for loop below, so it only loops through `m.rows` once.

Reply:
I think this is more readable and not bringing any performance issue.
The second filter (`HideDeleted`) is applied to the set of rows, which is the result of the first filter (`FilterText`). If the second filter is not enabled, then return the rows that remain after applying the first filter.